components/gucharmap/Makefile

```diff
@@ -49,9 +49,9 @@ CONFIGURE_OPTIONS+=	--localstatedir=/var/lib
 COMPONENT_INSTALL_ENV+=	GCONF_DISABLE_MAKEFILE_SCHEMA_INSTALL=1
```

Comment:
Please, if you touch Makefile, increment COMPONENT_REVISION, move it to appropriate category, remove BUILD_PKG_DEPENDS and depend.mk, add REQUIRED_PACKAGES